lux-workspace-hack/Cargo.toml

```diff
@@ -46,8 +46,6 @@ smallvec = { version = "1", default-features = false, features = ["const_new"] }
 time = { version = "0.3", default-features = false, features = ["formatting", "macros", "parsing"] }
 tokio = { version = "1", features = ["full"] }
 tokio-util = { version = "0.7", features = ["codec", "io"] }
-toml = { version = "0.8" }
-toml_edit = { version = "0.22", features = ["serde"] }
```

Comment:
question: is this removal supposed to happen?

Reply:
This was generated by `cargo hakari generate`.